tests/test_schema.py

```diff
@@ -4025,13 +4025,13 @@ def test_authentication(self):
 
 
 class TestMergeIdentitiesMutation(django.test.TestCase):
```

Comment:
Following what I said in another comments, maybe this mutation and its tests should be renamed too.